torch_xla/csrc/tensor.h

```diff
@@ -430,6 +430,10 @@ class XLATensor {
   // unchanged input otherwise.
   static XLATensor squeeze(const XLATensor& input, int dim);
```

Comment:
Can we have a standard?
Even if ATEN level API comes in as int, lets unify to int64 when we enter tensor.cpp/h

Reply:
Done.